build/alpine/install-packages.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
 set -e
-set -o pipefail
```

Comment:
nit: can you put this back too. FYI, it had to be separate from line 3 due to sh rather than bash used for Alpine

Reply:
Added back.